services/motion/builtin/builtin.go

```diff
@@ -157,32 +185,42 @@ func (ms *builtIn) MoveSingleComponent(
 	extra map[string]interface{},
 ) (bool, error) {
 	operation.CancelOtherWithLabel(ctx, "motion-service")
-	logger := ms.r.Logger()
 
-	components := robot.AllResourcesByName(ms.r, componentName.ShortName())
-	if len(components) != 1 {
-		return false, fmt.Errorf("got %d resources instead of 1 for (%s)", len(components), componentName.ShortName())
+	_, allResources, err := ms.fsService.AllCurrentInputs(ctx)
+	if err != nil {
```

Comment:
[minor] Can't this all just reduce down to `arm.FromRobot(componentName.ShortName()]`?

Reply:
No, because there is no `Robot` to

Reply:
Ah yes I see now.  Still can be a little less verbose if you directly index the resources map with the `componentName.ShortName()` and cast it to an arm.

Reply:
Done